playbooks/templates/standalone_parameters.yaml.j2

```diff
@@ -33,3 +35,18 @@ parameter_defaults:
   OctaviaCaKeyPassphrase: "secrete"
   OctaviaAmphoraSshKeyFile: "{{ ansible_env.HOME }}/octavia.pub"
   OctaviaAmphoraImageFilename: "{{ ansible_env.HOME }}/amphora.qcow2"
+
+  NeutronPhysicalDevMappings: "{{ neutron_dev_physical_mappings }}"
+  NovaReservedHostMemory: 4096
+
+# The following kernelargs, need to be added to the host GRUB_CMDLINE_LINUX
+# and the host needs to be rebooted and the values cheked.
+#  #KernelArgs: "default_hugepagesz=1GB hugepagesz=1G hugepages=32 iommu=pt intel_iommu=on isolcpus=1-19,21-39"
+#  Still don't hav TunedProfile working correctly
+#  TunedProfileName: "cpu-partitioning"
+# The PCI passthrough access address should be provided by user as variable
+  NovaPCIPassthrough:
+    - address: "0000:04:00.1"
```

Comment:
The address needs to be passed in as a variable. The value is obtained by looking at which sr-iov supporthing nic will be used under the the system menu. But user should provide it.